Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -712,6 +747,55 @@ def get_service_keys():
     }
 
 
+def add_responders_to_incident(incident_id, message, user_requests="", escalation_policy_requests="", requestor_id=DEFAULT_REQUESTOR):
+    """Add the responders to an incident"""
+    if RequestorID is None or RequestorID == "":
+        RequestorID = DEFAULT_REQUESTOR
+    url = SERVER_URL + RESPONDER_REQUESTS_SUFFIX.format(IncidentID)
+    body = {
+        'requestor_id': RequestorID,
+        'message': Message,
+        'responder_request_targets': []
+    }
+    for user_id in UserRequests.split(","):
+        body['responder_request_targets'].append({
+            'responder_request': {
+                "id": user_id,
+                "type": 'user_reference'
+            }
+        })
+    for escalation_policy_id in EscalationPolicyRequests:
+        body['responder_request_targets'].append({
+            'responder_request': {
+                "id": escalation_policy_id,
+                "type": 'escalation_policy_reference'
+            }
+        })
+    response = http_request('POST', url, data=body)
+    return extract_responder_request(response)
+
+
+def run_response_play(incident_id, from_email, response_play_uuid):
+    """Add the responders to an incident"""
+    url = SERVER_URL + RESPONSE_PLAY_SUFFIX.format(response_play_uuid)
+    body = {
+        'incident': {
+            'id': incident_id,
+            'type': 'incident_reference'
+        }
+    }
+    extra_headers = {
+        "From": from_email
+    }
+    response = http_request('POST', url, json_data=body, additional_headers=extra_headers)
+    if response != {"status": "ok"}:
+        raise Exception("Status NOT Ok - {}".format(response))
+    return CommandResults(
+        readable_output="Command ran successfully",
```

Comment:
Not a good readable output in my opinion, please be more specific

Reply:
Changed made in https://github.com/demisto/content/pull/17745/commits/072e7428c67b5d89c3c1bb6f178b2ac1116c04f4